CMakeLists.txt

```diff
@@ -174,7 +174,7 @@ if(NIFTI_INSTALL_EXPORT_NAME EQUAL "NIFTITargets")
 
   export(EXPORT NIFTITargets
         FILE "${CMAKE_CURRENT_BINARY_DIR}/NIFTI/NIFTITargets.cmake"
-        NAMESPACE ${NIFTI_PACKAGE_PREFIX}NIFTI::
```

Comment:
This is needed for incorporating NIFTI in other packages natively to avoid namespace collisions.  I really do not want to have to manually add this back in for ITK.

Reply:
Described below, I think I have now fixed this